tests/Hl7ListenerTrait.php

```diff
@@ -67,22 +67,19 @@ public function createTcpServer(int $port, int $totalClientsToConnect): void
         }
 
         if (($ret = socket_bind($socket, "localhost", $port)) === false) {
-            throw new RuntimeException('socket_bind() failed: reason: ' . socket_strerror($ret) . "\n");
+            throw new RuntimeException('socket_bind() failed: reason: ' . socket_strerror(socket_last_error($socket)) . "\n");
         }
         if (($ret = socket_listen($socket, 5)) === false) {
-            throw new RuntimeException('socket_listen() failed: reason: ' . socket_strerror($ret) . "\n");
+            throw new RuntimeException('socket_listen() failed: reason: ' . socket_strerror(socket_last_error($socket)) . "\n");
```

Comment:
In both of these `$ret` is a bool, and `socket_strerror()` expects an int. So, this calls `socket_last_error($socket)` instead.